dali/kernels/reduce/reduce_axes_gpu_test.cu

```diff
@@ -350,6 +349,56 @@ TYPED_TEST(ReduceMiddleGPUTest, ReduceOuter_B1_16M_3) {
 }
 #endif
 
+TEST(ReduceSamples, Sum) {
+  std::mt19937_64 rng(12345);
+  TestTensorList<float, 2> in, out;
+  TensorShape<2> sample_shape = { 480, 640 };
+#ifdef NDEBUG
```

Comment:
Why do we want to test differently in DEBUG?

Reply:
The reference implementation is really slow in debug - on the other hand, the smaller size is insufficient to give meaningful perf numbers.